src/DogStatsd.php

```diff
@@ -71,6 +75,8 @@ public function __construct(array $config = array())
         $this->apiKey = isset($config['api_key']) ? $config['api_key'] : null;
         $this->appKey = isset($config['app_key']) ? $config['app_key'] : null;
 
+        $this->globalTags = isset($config['global_tags']) ? $config['global_tags'] : [];
```

Comment:
@Firehed 

As you can see in [`composer.json`](), this library supports PHP 5.3, and thus, cannot work with short array syntax.

To fix the failing build, the following patch could be applied:

```diff
diff --git a/src/DogStatsd.php b/src/DogStatsd.php
index ca6a839..927f551 100644
--- a/src/DogStatsd.php
+++ b/src/DogStatsd.php
@@ -75,7 +75,7 @@ class DogStatsd
         $this->apiKey = isset($config['api_key']) ? $config['api_key'] : null;
         $this->appKey = isset($config['app_key']) ? $config['app_key'] : null;

-        $this->globalTags = isset($config['global_tags']) ? $config['global_tags'] : [];
+        $this->globalTags = isset($config['global_tags']) ? $config['global_tags'] : array();

         if ($this->apiKey !== null) {
             $this->submitEventsOver = 'TCP';
@@ -213,7 +213,7 @@ class DogStatsd
         if (!$all_tags) {
             return '';
         }
-        $tag_strings = [];
+        $tag_strings = array();
         foreach ($all_tags as $tag => $value) {
             if ($value === null) {
                 $tag_strings[] = $tag;
```